guides/source/active_record_querying.md

```diff
@@ -468,6 +468,17 @@ Another example would be if you wanted multiple workers handling the same
 processing queue. You could have each worker handle 10000 records by setting the
 appropriate `:start` and `:finish` options on each worker.
 
+**`:order_column`**
+
+By default `find_each` sorts by the primary key but some tables could have a UUID as primary key which makes this
+method useless. Allowing passing `:sort_column` will allow your to order by a integer column.
```

Comment:
Nit: suggestion for cleaning up the wording.

```suggestion
By default `find_each` sorts by the primary key, which might not always be appropriate (e.g. if your database uses UUID primary keys). `:cursor_column` can be passed to order the results by an integer column.
```

Reply:
Also, question: does it need to be an integer column? Theoretically a string cursor (or some other data type) could be used, right?

Reply:
To be honest I'm not completely sure, I know the query uses comparison operators like `>` or `<` but after looking for info in postgresql documentation as example they say this:

> These comparison operators are available for all built-in data types that have a natural ordering, including numeric, string, and date/time types. In addition, arrays, composite types, and ranges can be compared if their component data types are comparable.

So I guess there is no need that the cursor column has to be an integer.

Reply:
I've changed a bit the wording, tell me what you think.